plugins/argocd/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@janus-idp/backstage-plugin-argocd",
```

Comment:
I notice this is different from the existing road plugin `backstage-plugin-argo-cd` and I like that it's different as it will allow us to differentiate one from the other in dependabot configuration.